frontend/src/app/modules/interventions/details/details.component.html

```diff
@@ -18,22 +18,44 @@
     </button>
   </mat-card-actions>
 
-  <mat-list>
-    <mat-list-item> <mat-icon>done</mat-icon> {{ intervention.status | ekoInterventionStatus }} </mat-list-item>
-    <mat-list-item> <mat-icon>account_circle</mat-icon> {{ intervention.fullName || 'Brak' }} </mat-list-item>
-    <mat-list-item> <mat-icon>phone</mat-icon> {{ intervention.phoneNumber }} </mat-list-item>
-    <mat-list-item> <mat-icon>email</mat-icon> {{ intervention.email || 'Brak' }} </mat-list-item>
-    <mat-list-item> <mat-icon>location_on</mat-icon> {{ intervention.address }} </mat-list-item>
-  </mat-list>
-
   <mat-card-content>
-    {{ intervention.description }}
+    <section>
+      <h2 class="mat-h2">Podstawowe informacje</h2>
+      <mat-list>
+        <mat-list-item> <mat-icon>done</mat-icon> {{ intervention.status | ekoInterventionStatus }} </mat-list-item>
+        <mat-list-item> <mat-icon>account_circle</mat-icon> {{ intervention.fullName || 'Brak' }} </mat-list-item>
+        <mat-list-item> <mat-icon>phone</mat-icon> {{ intervention.phoneNumber }} </mat-list-item>
+        <mat-list-item> <mat-icon>email</mat-icon> {{ intervention.email || 'Brak' }} </mat-list-item>
+        <mat-list-item> <mat-icon>location_on</mat-icon> {{ intervention.address }} </mat-list-item>
+      </mat-list>
+    </section>
+
+    <section>
+      <h2 class="mat-h2">Opis sprawy</h2>
+      {{ intervention.description }}
+    </section>
+
+    <section>
+      <h2 class="mat-h2">Załączniki</h2>
+      <!-- TODO:  Temporary solution - UX-friendly attachment list with download option needed in the near future -->
+      <p *ngIf="!intervention.attachments || !intervention.attachments.length">Brak załączników</p>
+      <p *ngFor="let file of intervention.attachments">
```

Comment:
I have empty list of attachments here. Attachments are just ids, without id field
![image](https://user-images.githubusercontent.com/119451/81009947-ce233680-8e55-11ea-8b99-17a8b2db8d14.png)

Reply:
A, I see, it will work after merging this one probably
https://github.com/lightning-team/Ekostraz/pull/105

Reply:
Yes, you are right.
If you really want to test it now, then you should probably create a separate branch locally, merge changes from this PR and #105 together and run your local server. Then it would work only for the new files added (old ones don't have a proper Content-Disposition set)